packages/sdks/nextjs-sdk/src/server/authMiddleware.ts

```diff
@@ -105,13 +105,13 @@ const addSessionToHeadersIfExists = (
 // if the user is not authenticated, it redirects to the redirectUrl
 // if the user is authenticated, it adds the session to the headers
 const createAuthMiddleware =
-	(options: MiddlewareOptions = {}) =>
-	async (req: NextRequest) => {
+  (options: MiddlewareOptions = {}) =>
+  async (req: NextRequest, next?: () => Promise<NextResponse>) => {
```

Comment:
in the example, I don't see we pass next, I'm not sure this is how things are chained today in next